cognee/modules/pipelines/operations/run_tasks.py

```diff
@@ -190,7 +195,6 @@ async def run_tasks_with_telemetry(tasks: list[Task], data, pipeline_name: str):
 
 async def run_tasks(tasks: list[Task], data = None, pipeline_name: str = "default_pipeline"):
     config = get_current_settings()
```

Comment:
Let's move also this config inside `run_tasks_with_telemetry`, then we don't have to pass it in.

Reply:
done